qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -25,33 +25,41 @@
 
 
 class T2Ramsey(BaseExperiment):
+    r"""T2Ramsey Experiment.
 
-    r"""
-    T2Ramsey Experiment
+    # section: overview
 
-    Overview
         This experiment is used to estimate two properties for a single qubit:
         T2* and Ramsey frequency.
+
+        See `Qiskit Textbook <https://qiskit.org/textbook/ch-quantum-hardware/\
+        calibrating-qubits-pulse.html>`_  for a more detailed explanation on
+        these properties.
+
         This experiment consists of a series of circuits of the form
 
-    .. parsed-literal::
+        .. parsed-literal::
+
+               ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐
+          q_0: ┤ H ├┤   DELAY(t)   ├┤ P(λ) ├─░─┤ H ├─░─┤M├
+               └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘
+          c: 1/═════════════════════════════════════════╩═
+                                                        0
```

Comment:
```suggestion
                   ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐

              q_0: ┤ H ├┤   DELAY(t)   ├┤ P(λ) ├─░─┤ H ├─░─┤M├

                   └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘

              c: 1/═════════════════════════════════════════╩═

                                                            0
```
I think you need extra indent for the parsed-literal content.